tests/unit/h/services/group_members_test.py

```diff
@@ -143,7 +141,6 @@ def test_it_proxies_to_member_join_and_leave(
         )
         assert sorted(group_members_service.member_leave.call_args_list) == sorted(
             [
-                mock.call(group, group.creator.userid),
```

Comment:
This is just testing that when `update_members()` is called it calls `member_leave()` for any pre-existing members that aren't in the new list of members. Since the factory was adding the creator as a member, this call was being made. But it's actually irrelevant to the test and can now be removed.